README.md

```diff
@@ -24,15 +24,15 @@ The `create_services.sh` uses:
 Before starting it, please install the required dependences, login with the [Aiven CLI](https://github.com/aiven/aiven-client) and execute:
```

Comment:
Might be worth explaining that one can use `pip install -r aiven_metadata_parser/requirements.txt` to install the Python packages (perhaps in the list item that mentions that file)

Reply:
I think it's mentioned in line #46 already